app/src/main/java/com/spartahack/spartahack17/Fragment/ScheduleFragment.java

```diff
@@ -1,101 +1,89 @@
 package com.spartahack.spartahack17.Fragment;
 
-
 import android.os.Bundle;
+import android.support.annotation.NonNull;
 import android.support.v4.app.Fragment;
 import android.support.v7.widget.LinearLayoutManager;
 import android.support.v7.widget.RecyclerView;
-import android.util.Log;
 import android.view.View;
 
 import com.spartahack.spartahack17.Adapters.EventListAdapter;
 import com.spartahack.spartahack17.Adapters.SimpleSectionedRecyclerViewAdapter;
 import com.spartahack.spartahack17.Model.Event;
+import com.spartahack.spartahack17.Presenter.SchedulePresenter;
 import com.spartahack.spartahack17.R;
-import com.spartahack.spartahack17.Retrofit.GSONMock;
-import com.spartahack.spartahack17.Retrofit.ParseAPIService;
+import com.spartahack.spartahack17.View.ScheduleView;
 
 import org.joda.time.DateTime;
 import org.joda.time.DateTimeComparator;
 
 import java.text.SimpleDateFormat;
 import java.util.ArrayList;
-import java.util.Collections;
 import java.util.Locale;
 
-import butterknife.BindView;
-import rx.Subscriber;
-import rx.android.schedulers.AndroidSchedulers;
-import rx.schedulers.Schedulers;
+import butterknife.Bind;
 
 /**
  * A simple {@link Fragment} subclass.
  */
-public class ScheduleFragment extends BaseFragment {
+public class ScheduleFragment extends MVPFragment<ScheduleView, SchedulePresenter> implements ScheduleView {
+
+    private static final String TAG = "ScheduleFragment";
 
     /** Recycler view that displays all objects */
-    @BindView(android.R.id.list) RecyclerView recyclerView;
+    @Bind(android.R.id.list) RecyclerView recyclerView;
```

Comment:
this will have to be bindview, i would reccomend rebasing your code onto develop